### PR TITLE
fix some errors

### PR DESCRIPTION
**Pull request by AntonioMirarchi**
- [x] move `batch_comp`  from global variable to function args
- [x] include `repulsion `and `repulsioncg `in build_parameters function. I removed 'LJ' since we have term.lower() for all the terms.

### CODE REVIEW

File: torchmd/run.py [hunk 1]
```diff
@@ -18,7 +18,6 @@
 from utils import converter_xyz_output
 
 FS2NS = 1e-6
-batch_comp = False
 # set to false by default, it become True when you use the .npz file
```
Comment (RaulPPelaez) [inline torchmd/run.py:21]: Should you remove this comment?

File: torchmd/run.py [hunk 2]
```diff
@@ -150,7 +149,7 @@ def get_args(arguments=None):
 precisionmap = {"single": torch.float, "double": torch.double}
 
 
-def setup(args):
+def setup(args, batch_comp=None):
```
Comment (RaulPPelaez) [inline torchmd/run.py:152]: batch_comp used to be false by default, now it is None, is this really what you want?

File: torchmd/parameters.py [hunk 3]
```diff
@@ -117,7 +117,7 @@ def build_parameters(self, ff, mol, terms):
             self.masses = mol.masses
         else:
             self.masses = self.make_masses(ff, mol.atomtype)
-        if "lj" in terms or "LJ" in terms:
+        if any(elem in terms for elem in ["lj", "repulsioncg", "repulsion"]):
```
Comment (RaulPPelaez) [inline torchmd/parameters.py:120]: You should also include "LJ", right?

File: torchmd/run.py (hunk 2 above)
Comment (AntonioMirarchi) [reply torchmd/run.py:152]: it works fine in any case, but I will set to false, makes more sense!

**Review by RaulPPelaez — APPROVED** [review]
LGTM. @stefdoerr please merge as you see fit.

**Review by stefdoerr — APPROVED** [review]

**Pull request merged**